qiskit/primitives/sampler_result.py

```diff
@@ -25,10 +25,14 @@
 class SamplerResult:
     """
     Result of Sampler
+
+    .. code-block:: python
+
+        result = sampler(params, grouping)
+
+    where the i-th elements of `result` correspond to the expectation using the circuit and
+    observable given by `grouping[i]` and the parameters bounds by `params[i]`
     """
 
     quasi_dists: list[QuasiDistribution]
     metadata: list[dict[str, Any]]
-
```

Comment:
I agree with the removal for simplicity. However, since it is used in the document, the document needs to be fixed if it is to be removed. (we can change the document after this PR is merged)